lua/nvim-navbuddy/actions.lua

```diff
@@ -1,8 +1,15 @@
+local USER_FOLDMETHOD = vim.o.foldmethod
+local USER_FOLDMARKER = vim.o.foldmarker
+local USER_FOLDMARKER_O, USER_FOLDMARKER_C = USER_FOLDMARKER:match("([^,]+),([^,]+)")
+local CMD_FOLD_N_ZF = "normal! zf"
+local CMD_FOLD_N_ZD = "normal! zd"
```

Comment:
No need of this? can be done inside the action function.

Reply:
yes I know =}

As the command is used 2 times, if in the futur, it is changed, I only have to change one time...

Obviously our style is different. I dont like duplicated code, that's why...